3rd-party/vaultpress.php

```diff
@@ -29,6 +46,7 @@ function jetpack_vaultpress_rewind_enabled_notice() {
  * @since 5.8
  */
 function jetpack_vaultpress_rewind_check() {
+	add_action( 'admin_notices', 'jetpack_vaultpress_rewind_enabled_notice' );
```

Comment:
I haven't tested, but I think this line needs to be removed before merging.

Reply:
Took care of that in fc630c4156bac7c907ad999e5b90ee59c6e76f0d